Echo/EchoNode.cpp

```diff
@@ -0,0 +1,246 @@
+#include "EchoNode.hpp"
+#include <iostream>
+#include <fstream>
+
+
+using namespace std;
+
+
+namespace polysync
+{
+
+
+void PolySyncEcho::initStateEvent()
+{
+    if ( _filteredForSingleMsgFlag )
+    {
+        try
+        {
+            registerListener( getMessageTypeByName ( _msgName ) );
+        }
+
+        catch ( ... )
+        {
+            std::cout << "\nPlease enter a valid PS message type. \n";
+
+            printAvailableMessage( getAvailableMessageNames() );
+
+            disconnectPolySync();
+        }
+    }
+
+    else
+    {
+        registerListenerToAllMessageTypes();
+    }
+}
+
+
+void PolySyncEcho::setConfigurationEvent( int argc, char * argv [] )
+{
+    std::string filterFlag = "-filterMessage";
```

Comment:
maybe this should just be -filter -filterMessage is a little cumbersome and easy for me to mistype.